gbpservice/nfp/configurator/agents/agent_base.py

```diff
@@ -109,8 +109,8 @@ def _notification(self, data):
         """
 
         event = self.sc.new_event(
-            id='NOTIFICATION_EVENT', key='NOTIFICATION_EVENT', data=data)
-        self.sc.poll_event(event)
+            id='STASH_EVENT', key='STASH_EVENT', data=data)
```

Comment:
We're presently importing from generic config constants file which should not be the case as this is a common file. So, can I replace the current constant file with the common constant file?